fetch.bs

```diff
@@ -3684,11 +3785,35 @@ steps:
 
  <li><p>Wait for either <var>response</var>'s <a for=response>body</a> to be null, or
  <var>response</var>'s <a for=response>body</a>'s <a for=body>stream</a> to be
- <a for=ReadableStream>closed</a> or <a for=ReadableStream>errored</a>, and then set
- <var>request</var>'s <a>done flag</a>.
- <!-- This is really bad and needs to be handled differently at some point. -->
+ <a for=ReadableStream>closed</a> or <a for=ReadableStream>errored</a>, and perform the following steps:
+ <ol>
+  <li><p>Set <var>request</var>'s <a>done flag</a>.
+   <!-- This is really bad and needs to be handled differently at some point. -->
+  <li>Call <a for=/>report resource timing</a> with <var>fetchParams</var> and <var>response</var>.
+ </ol>
 </ol>
 
+<p>To <dfn>report resource timing</dfn>, given <a for=/>fetch params</a> <var>fetchParams</var> and
+<a for=/>response</a> <var>response</var>, perform the following steps:
+<ol>
+  <li>Let <var>globalObject</var> be <var>fetchParams</var>'s <a for="fetch params">request</a>'s
+  <a for=request>client</a>'s <a for="environment settings object">global object</a>.
+  <li>Let <var>timingInfo</var> be <var>fetchParams</var>'s <a for="fetch params">timing info</a>.
+  <li>Let <var>requestedURL</var> be <var>response</var>'s <a for=response>URL list</a>'s first item.
+  <li>If <var>response</var>'s <a for=response>timing allow passed flag</a> is not set, set
```

Comment:
Do we still need this flag on response?

Reply:
I will remove it once it's fully integrated with resource-timing, in the meantime there is some duplication